s_vae/models/s_vae/vMF.py

```diff
@@ -125,23 +125,24 @@ def __Reflect(self, H, x):
         x (Tensor): vector to transform
         """
 
-        return torch.mv(H,x.view(-1).float())
-
+        return torch.mv(H, x.view(-1).float())
 
     def entropy(self):
+        # Todo: Move Bessel function to GPU
         output = (
-            self.scale
-            * Bessel(self.ndim/ 2, self.scale)
-            / Bessel((self.ndim/ 2) - 1, self.scale)
+                self.scale
+                * Bessel(self.ndim / 2, self.scale).to(self.device)
+                / Bessel((self.ndim / 2) - 1, self.scale).to(self.device)
         )
-        
-        return output+ self._log_normalization()
+
+        return output + self._log_normalization()
 
     def _log_normalization(self):
+        # Todo: Move Bessel function to GPU
```

Comment:
This is done, right?

Reply:
Kind of. The Bessel is still on the CPU,but the result is transferred to the GPU if needed. We can remove the TODO, I just forgot it